src/test/java/com/hpe/adm/octane/ideplugins/integrationtests/IntegrationTestBase.java

```diff
@@ -116,7 +122,8 @@ public Long createWorkSpace() {
             response = octaneHttpClient.execute(postNewWorkspaceRequest);
 
         } catch (Exception e) {
-            e.printStackTrace();
+            logger.error("Error while trying to get the response when creating a new workspace!");
+            Assert.fail(e.toString());
```

Comment:
Also logger can take a throwable as argument, might aswell add it, as the second param